lib/ui/semantics.dart

```diff
@@ -307,6 +309,24 @@ class SemanticsFlag {
   /// is a password or contains other sensitive information.
   static const SemanticsFlag isObscured = const SemanticsFlag._(_kIsObscuredIndex);
 
+  /// Whether the semantics node is the root of a subtree for which values
+  /// should be announced.
+  /// 
+  /// When a semantics node with an edge flag is added to the tree, the
+  /// framework will search for a child with a route flag and non-empty
+  /// semantic value and announce it as an edge transition.
+  /// 
+  /// This is used in widgets such as Routes, Drawers, and Dialogs to 
+  /// communicate significant changes in the visible screen.
+  static const SemanticsFlag isEdge = const SemanticsFlag._(_kIsEdgeIndex);
+
+  /// Whether the semantics node value is the name of a visually distinct edge.
+  /// 
+  /// This is used by certain widgets like Drawers and Dialogs, to indicate
+  /// that the node's semantic value can be used for an edge triggered
+  /// semantic update.
```

Comment:
maybe mention that changing the value is meaningless

Reply:
Maybe slightly more specific: ... changing the value **of the active edge** is meaningless....

Reply:
Fixed